hclasses/CTString.cpp

```diff
@@ -469,6 +469,25 @@ namespace CT {
     substringSelf(s, e + 1);
   }
 
+  void string::trimWhiteSpacesAndLinesSelf() {
+
+    int s = -1, e = privatelength;
+    const char *value = useStack ? stackValue : heapValue;
+    for (size_t j = 0; j < privatelength; j++) {
+      if (value[j] != ' ' || value[j] != 10 || value[j] != 13) {
```

Comment:
This should be `&&` instead of `||`? And it is not possible to use the char representation '\n' and '\r' for clarity?

Reply:
Yes you are right. I changed and now I get what I wanted. If you you place spaces or newlines in the configuration it does now not cause issues.